<?xml version="1.0"?>

<launch>

  <arg name="goal_topic" default="/move_base_simple/goal"/>


  <node name="goal_pub" pkg="flight_camp" type="goal_pub.py" output="screen">

    <param name="goal_topic" value="$(arg goal_topic)"/>

  </node>



</launch>

